src/geophires_x/CylindricalReservoir.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, model: Model):
 
         self.InputDepth = self.ParameterDict[self.InputDepth.Name] = floatParameter(
             "Cylindrical Reservoir Input Depth",
+            value=3.0,
```

Comment:
Remove - only `DefaultValue` should be specified, not `value` (which is redundant with `DefaultValue` in this case anyway)